Products/ZenModel/DeviceOrganizer.py

```diff
@@ -509,11 +509,28 @@ def setManageIp(self, REQUEST=None):
         if REQUEST:
             return self.callZenScreen(REQUEST)
 
-    def collectDevice(self, REQUEST=None):
+    def collectDevice(self, REQUEST=None, write=None, debug=False):
         """model all devices in this Organizer.
         """
-        [ d.collectDevice() for d in self.getSubDevices() ]
-        if REQUEST:
+        subDevices = [d for d in self.getSubDevices()]
+        if not subDevices and write:
+            write("Not found devices in organizer")
+        else:
```

Comment:
The `else:` clause is unnecessary.  Iterating over an empty list behaves like a no-op.